Makefile

```diff
@@ -32,7 +32,7 @@ clean: pyclean docsclean
 	@rm -rf .tox
 
 venv:
-	@virtualenv -p python2.7 venv
+	@virtualenv -p python3.5 venv
```

Comment:
from python3.6 at least it's recommended to create virtualenv with the `venv` module: https://docs.python.org/3.6/whatsnew/3.6.html#id8, this also works with python3.5:

`python3.5 -m venv venv`

While we're talking, why not use python3.6 instead of python3.5?